examples/generation_api/custom_model.py

```diff
@@ -0,0 +1,41 @@
+# Example: Generate (2~7) random numbers using Llama3.2 model served by ollama
+# To run this example, you can either
+# 1. Use the sotopia devcontainer and run the following command in the terminal:
+#        poetry run python examples/generation_api/custom_model.py
+#    This example can also serve a sanity check for your devcontainer setup.
+# OR 2. after installing sotopia, install ollama, and then:
```

Comment:
I don't think it's clear to me how we connect with the customized model here. Can we add things to the doc?

Reply:
It is set up directly in devcontainer